src/main.py

```diff
@@ -175,12 +187,32 @@ def cli():
 @cli.command()
 @option_disable_ssl_verification
 @click.argument("config", type=click.File("r"))
-def sync(config, disable_ssl_verification):
+@click.option("--project-id", type=int, help="The ID of your dbt Cloud project.")
+@click.option("--environment-id", type=int, help="The ID of your dbt Cloud environment.")
+def sync(config, project_id, environment_id, disable_ssl_verification):
     """Synchronize a dbt Cloud job config file against dbt Cloud.
 
     CONFIG is the path to your jobs.yml config file.
     """
-    change_set = build_change_set(config, disable_ssl_verification)
+    cloud_project_id = 0
+    cloud_environment_id = 0
+
+    # If project-id is provided, environment_id should also be provided as an argument, and vice versa
+    if project_id and environment_id is None:
+        raise click.BadParameter("Both --project-id and --environment-id must be provided")
+    if project_id is None and environment_id:
+        raise click.BadParameter("Both --project-id and --environment-id must be provided")
```

Comment:
I don't know if I agree with the logic/requirement here. 

For me, we could provide a list of project ids OR a list of env ids but wouldn't necessarily need both.
When we provide and env id, we are already filtering by project id.

If/when we select both, we should agree on the behaviour:
1. (my preference) if project id and environments id are set, we sync/plan/... for the union of all those. i.e. all the environments of the project selected AND the individual environments selected 
2. if project id and environments id are set, we select the intersection of both (but in that case the project id is not needed as jobs can be filtered by env id only)

Reply:
As mentioned above, I feel we should make project ID a mandatory CLI argument for the commands. Environment ID can be optional.